Kitodo/src/main/java/org/kitodo/services/workflow/WorkflowControllerService.java

```diff
@@ -49,6 +50,7 @@
 
 public class WorkflowControllerService {
 
+    private final MetadatenSperrung msp = new MetadatenSperrung();
```

Comment:
Please use meaningful variable names. 
The class "metadatensperrung" should be translated.